MockMDS/src/main/java/io/mosip/mock/sbi/service/SBIServiceResponse.java

```diff
@@ -135,26 +135,32 @@ public String processDeviceDicoveryInfo (SBIMockService mockService)
              List<DiscoverDto> infoList = new ArrayList<DiscoverDto> ();
              if (type == null || type.trim().length() == 0)
     		 {
+            	 /*
             	 DiscoverDto discoverInfo = new DiscoverDto ();
             	 Map<String, String> errorMap  = new HashMap<String, String>() {{
             		    put("502",  SBIJsonInfo.getErrorDescription(lang, "502"));
             		}};
             	 discoverInfo.error = errorMap;
             	 infoList.add(discoverInfo);
             	 return objectMapper.writeValueAsString(infoList);
+            	 */
+            	 return SBIJsonInfo.getErrorJson (lang, "502", "");
     		 }
              else if (!type.equalsIgnoreCase(ApplicationPropertyHelper.getPropertyKeyValue(SBIConstant.MOSIP_BIOMETRIC_TYPE_BIOMTRIC_DEVICE).trim().toLowerCase())
         		 && !type.equalsIgnoreCase(ApplicationPropertyHelper.getPropertyKeyValue(SBIConstant.MOSIP_BIOMETRIC_TYPE_FINGER).trim().toLowerCase())
         		 && !type.equalsIgnoreCase(ApplicationPropertyHelper.getPropertyKeyValue(SBIConstant.MOSIP_BIOMETRIC_TYPE_FACE).trim().toLowerCase())
         		 && !type.equalsIgnoreCase(ApplicationPropertyHelper.getPropertyKeyValue(SBIConstant.MOSIP_BIOMETRIC_TYPE_IRIS).trim().toLowerCase()))
              {
+            	 /*
```

Comment:
Please remove the commented code we do not need them. We can look at the history if required.

Reply:
Removed all the comments